task/upload_test.go

```diff
@@ -0,0 +1,116 @@
+package task
+
+import (
+	"github.com/livepeer/catalyst-api/video"
+	"github.com/livepeer/livepeer-data/pkg/data"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestToTranscodeFileTaskOutput(t *testing.T) {
+	tests := []struct {
```

Comment:
Can you add a case without MP4 outputs as well?

Reply:
Good idea, added.